coda/coda_mdstore/mdstore_oai.py

```diff
@@ -95,14 +94,14 @@ def listIdentifiers(self, metadataPrefix=None, from_=None, until=None,
     def listRecords(self, metadataPrefix=None, from_=None, until=None,
                     set=None, cursor=0, batch_size=10):
 
-        return self.listStuff(
+        return self.makeList(
             metadataPrefix=metadataPrefix, from_=from_,
             until=until, set=set, cursor=cursor, batch_size=batch_size,
             headersOnly=False
         )
 
-    def listStuff(self, metadataPrefix=None, from_=None, until=None,
-                  set=None, cursor=0, batch_size=10, headersOnly=True):
+    def makeList(self, metadataPrefix=None, from_=None, until=None,
```

Comment:
I agree; if we are changing the name from `listStuff`, more description would be worthwhile.